schema.graphql

```diff
@@ -1,3 +1,5 @@
+# Credit to https://github.com/OpenZeppelin/openzeppelin-subgraphs, included under MIT License
```

Comment:
Hmm, I know it might be some repetitiveness, but I wonder if we should include this statement only above each specific relevant types / code blocks. Right now it looks more like the whole schema is from OZ.

Or is there a way you might structure it so that it's clear what we're borrowing and it's clear what we wrote ourselves re: Option domain-specific code?

Reply:
how about this? https://github.com/valorem-labs-inc/valorem-subgraph/pull/48/commits/1f2fdbe0c2a0b855d3fd26ddcb86c78da90d45b1